README.md

```diff
@@ -24,7 +24,7 @@ Name     | Description | OS
 arp | Exposes ARP statistics from `/proc/net/arp`. | Linux
 conntrack | Shows conntrack statistics (does nothing if no `/proc/sys/net/netfilter/` present). | Linux
 cpu | Exposes CPU statistics | Darwin, Dragonfly, FreeBSD, Linux
-diskstats | Exposes disk I/O statistics from `/proc/diskstats`. | Linux
+diskstats | Exposes disk I/O statistics from `/proc/diskstats`. | Darwin, Linux
```

Comment:
That description is not longer correct, Darwin doesn't have a proc filesystem. Either drop it or describe each source.